market-generator/src/main/java/com/market/banica/generator/configuration/MarketConfigurationImpl.java

```diff
@@ -0,0 +1,167 @@
+package com.market.banica.generator.configuration;
+
+import com.market.banica.generator.exception.NotFoundException;
+import com.market.banica.generator.model.GoodSpecification;
+import com.market.banica.generator.property.LinkedProperties;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.EnableMBeanExport;
+import org.springframework.jmx.export.annotation.ManagedOperation;
+import org.springframework.jmx.export.annotation.ManagedResource;
+import org.springframework.stereotype.Component;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Properties;
+import java.util.concurrent.locks.ReadWriteLock;
+import java.util.concurrent.locks.ReentrantReadWriteLock;
+import java.util.function.BiConsumer;
+
+@Component
+@ManagedResource
+@EnableMBeanExport
```

Comment:
Move this annotation in configuration class or in the boot class as it enables default exporting of all standard MBeans from the Spring context, as well as well all @ManagedResource annotated beans.